realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -248,3 +250,15 @@ JNIEXPORT void JNICALL Java_io_realm_internal_LinkView_nativeRemoveTargetRow
         return lvr->remove_target_row( S(pos) );
     } CATCH_STD()
 }
+
+static void finalize_link_view(jlong ptr)
+{
+    TR_ENTER_PTR(ptr)
```

Comment:
I would prefer to have the `TR_ENTER` and `TR_ENTER_PTR` macros at the JNI border. The macros writes the function's name to the log, and `finalize_link_view` provide us with less information compare to, say, `Java_io_realm_internal_LinkView_nativeClose`.